website/conferences/views.py

```diff
@@ -96,22 +95,16 @@ def add_poster_by_email(conference, message):
         else:
             set_password_url = None
 
-        node, node_created = Node.objects.get_or_create(
-            title__iexact=message.subject,
-            is_deleted=False,
-            _contributors__guids___id=user._id,
-            defaults={
-                'title': message.subject,
-                'creator': user
-            }
+        # Always create a new meeting node
+        node = Node.objects.create(
+            title=message.subject,
+            creator=user
         )
-        if node_created:
-            nodes_created.append(node)
-            node.add_system_tag('osf4m')
-            node.save()
+        node.add_system_tag('osf4m')
+        node.save()
 
         utils.provision_node(conference, message, node, user)
-        utils.record_message(message, nodes_created, users_created)
+        utils.record_message(message, [node], users_created)
```

Comment:
Borderline out of scope, but it might be nice to clean this up and pass the objects themselves since `nodes_created` and `users_created` are always lists with 0-1 items. You'd have to change the body of `record_message` a bit, but this is the only place it's used so it's fairly low-risk. 

Not necessary, but was just a thought I had. Leaving it up to you 👍

Reply:
I'd definitely considered that! I'll go for it!